vmdb/app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -69,6 +69,28 @@ def show_container(record, controller_name, display_name)
         count_text = pluralize(@view.extras[:total_count] - @view.extras[:auth_count], "other #{title.singularize}")
         @bottom_msg = "* You are not authorized to view #{count_text} on this #{ui_lookup(:tables => @table_name)}"
       end
+    elsif @display == "container_replicators" || session[:display] == "container_replicators" && params[:display].nil?
```

Comment:
I think @kbrock is trying to simplify all this similar code with helpers, see #3343

Check if you can do the same here (anyway one of you will have to rebase probably). Thanks!